imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml

```diff
@@ -147,6 +147,9 @@ post:
 
         **UNSUPPORTED_KYC_REGION**
         The KYC region is not yet supported.
+
+        **KYC_STATEMENT_REGION_MISMATCH**
+        The latest unchecked KYC statement is for a region that is different from the requested region.
```

Comment:
just curious, what does "unchecked" mean in this context?

Reply:
Great question! By 'unchecked,' it means that the latest KYC statement hasn't been submitted yet or doesn't require manual review.  If that's the case and the region is different from the requested one, that's when we return the KYC_STATEMENT_REGION_MISMATCH error. Thanks for pointing that out! You're right—we should rephrase that part to make it clearer. Any thoughts @rb455 ?

Reply:
I guess it's a bit confusing cos I'm not very familiar with the domain and KYC statement lifecycle :) Do we create a blank KYC statement first and the attach supporting documents to it, and that's when it becomes "submitted"?
We don't need to reiterate it all here but adding a back-link to a corresponding KYC guide might be helpful

Reply:
I don't see any suggestion to rephrase it in Kira's comment, she is just curious, right?

'unchecked' means that the latest KYC statement from the partner hasn't been submitted to FrankieOne for a KYC verification yet

Reply:
'unchecked' means that the latest KYC statement from the partner hasn't gone through a KYC verification yet. Submitting a statement is separate from initiating the check.

Reply:
adding a backlink? i don't mind, won't hurt

Reply:
> 'unchecked' means that the latest KYC statement from the partner hasn't gone through a KYC verification yet. Submitting a statement is separate from initiating the check. The check is initiated by calling spending prerequisites

@rb455 thanks, that makes much more sense now.